frontend/src/actions/symptoms.js

```diff
@@ -0,0 +1,34 @@
+import { SUCCESS, FETCH_SYMPTOMS_START, FETCH_SYMPTOMS, SUBMIT_SYMPTOMS_START, SUBMIT_SYMPTOMS } from './types';
+import api from 'utils';
+import history from '../history';
+
+export const fetchSymptoms = () => async (dispatch) => {
+  dispatch({type: FETCH_SYMPTOMS_START});
+  const response = await api(`symptoms`);
+  dispatch({
+    type: FETCH_SYMPTOMS,
+    payload: {
+      status: response.error || { type: SUCCESS },
+      all: (!response.error && response) || [],
+    }
+  });
+}
+
+export const submitSymptoms = (symptoms, nextPath) => async (dispatch, getState) => {
+  dispatch({type: SUBMIT_SYMPTOMS_START});
+  const {story} = getState().story;
+  const response = await api(`stories/${story.id}/symptoms`, {
+    method: 'POST',
+    body: symptoms.map(id => ({
```

Comment:
I hear you, and it does make sense, but it was troublesome for the validation of data received in the backend. It involved creating a pydantic schema that fit this model, and it felt very out of place among the rest of the models.

While I'm open to reconsidering it, implementing it now would involve enough change that I'd rather not do it at this point (both for timing & because I'm not in the best place). Let's sit this one out and see how we feel about it as the app grows.